### PR TITLE
Enable a few env vars to make it easier to setup the docker-compose r…

### PR DESCRIPTION
**Pull request by ssallam**
…un for developers. This requires supporting changes in keeper-contracts (28507cf385a01cde0d87dc8e4a9b6b8da73f7727).

## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue https://github.com/oceanprotocol/ocean/issues/174

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

**Comment by ssallam** [comment]
Depends on https://github.com/oceanprotocol/keeper-contracts/pull/116

File: start_ocean_local_parity_node.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```
Comment (kremalicious) [inline start_ocean_local_parity_node.sh]: awesome, good call with some bash scripts. But these look like a lot of duplication and we could just use one script, `start_ocean.sh`, and do the rest with options, like:

```
start.sh
start.sh --no-pleuston
start.sh --local-parity
```

File: docker-compose.yml [hunk 2]
```diff
@@ -18,9 +18,14 @@ services:
     networks:
     - backend
     environment:
-      LOCAL_CONTRACTS: 'true'
+      LOCAL_CONTRACTS: "true"
```
Comment (kremalicious) [inline docker-compose.yml]: are we sure we need this as a string here instead of a boolean?

Comment (ssallam) [reply docker-compose.yml]: This is the check in keeper-contracts: `"${LOCAL_CONTRACTS}" = "true"
I didn't write this, originally.

Javier @jcortejoso, can this be a boolean?

File: start_ocean_local_parity_node.sh (hunk 1 above)
Comment (ssallam) [reply start_ocean_local_parity_node.sh]: Good point, so it is.

File: docker-compose.yml (hunk 2 above)
Comment (jcortejoso) [reply docker-compose.yml]: Yeah, `docker-compose` does not accept booleans.

File: start_ocean.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+export OCEAN_VERSION=latest
+# Must be set to true for the first run, change it to "false" to avoid migrating the smart contracts on each run.
+export DEPLOY_CONTRACTS="true"
+# Ganache specific option, these two options have no effect when not running ganache-cli
+export GANACHE_DATABASE_PATH="."
```
Comment (jcortejoso) [inline start_ocean.sh]: This folder reference to the $PWD where the user run the script from, not the script's folder. To get the script's folder `DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"` can be used.

File: start_ocean.sh [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+export REUSE_DATABASE="false"
+# Specify which ethereum client to run or connect to: kovan, ganache, or ocean_poa_net_local
+export KEEPER_NETWORK_NAME="ganache"
+export ARTIFACTS_FOLDER=~/.ocean/keeper-contracts/artifacts
```
Comment (jcortejoso) [inline start_ocean.sh]: This should be a relative path to the script's folder rather than an absolute path?

File: start_ocean.sh (hunk 3 above)
Comment (ssallam) [reply start_ocean.sh]: My intention was the CWD or $PWD.

File: start_ocean.sh (hunk 4 above)
Comment (ssallam) [reply start_ocean.sh]: This is in `.ocean` under the user's home folder.

File: docker-compose.yml (hunk 2 above)
Comment (kremalicious) [reply docker-compose.yml]: interesting, good learning, thanks

**Pull request merged**